internal/grpc/services/gateway/usershareprovider.go

```diff
@@ -733,3 +742,23 @@ func shareIsSpaceRoot(key *collaboration.ShareKey) bool {
 	}
 	return refIsSpaceRoot(key.ResourceId)
 }
+
+func isEqualGrantee(a, b provider.Grantee) bool {
```

Comment:
Would it make sense to have a and b being `*provider.Grantee`? We would avoid having to dereference them in line `706`.

Reply:
done